Gemfile

```diff
@@ -80,12 +80,12 @@ gem 'htmltoword'
 # gem 'debugger'
 
 group :development, :test do
```

Comment:
A suggestion would be to put a comment for the reason why you are pinning these gems to a certain version. For example what we do in HN: https://github.com/ualbertalib/HydraNorth/blob/master/Gemfile#L63 or look on line 6 of this file for what someone did for rubyzip: https://github.com/ualbertalib/DMPonline_v4/blob/9c66fbd0eeb7e4f51a389501fc1a772c08e1cc7d/Gemfile#L5

Also...considering these gems are under development/test groups, theses are not security concerns as these do not run in production.

My bigger concerns be why we using heavily gems like nokogiri (C compliled gem which needs libxml2 and libxslt among other things) for development/test?

Only place I see  mail/nokogiri being used is in a `user.rb` spec test. So perhaps...instead of bumping these...maybe we can reevalute this test. I am pretty sure you will quickly find it does NOT need these gems. (Even if it does, theres better solutions, for example ruby lib comes with its own XML parser natively instead of using nokogiri)